test/functionalTest/cases/1259_new_json_for_v2/put_attribute_value_type_check.test

```diff
@@ -0,0 +1,109 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Yes, this directory is where I found the test with 30 steps  (put_attribute_value.test)